### PR TITLE
disk_space: fix alert_absolute

### LINKED ISSUE #1182: alert_absolute in disk space is not working correctly, when true
I build latest master and use this config
```
[[block]]
block = "disk_space"
alert = 10.0
alert_absolute = true
format = "{path^2}{available;G}"
info_type = "available"
interval = 300
path = "/data"
unit = "GB"
warning = 50.0
```
Currently I have 34 GB available, but block stay the same color as it was before. If I change `alert_absolute` to false and use correct percentage for warning, the block styled as it should be styled for warning level. In my case I use dark-gruvbox theme and warning color is yellow

### PR DESCRIPTION
**Pull request by MaxVerevkin**
Closes #1182

### CODE REVIEW

File: src/blocks/disk_space.rs [hunk 1]
```diff
@@ -208,10 +208,10 @@ impl Block for DiskSpace {
         // Send percentage to alert check if we don't want absolute alerts
         let alert_val = if self.alert_absolute {
             (match self.unit {
-                Prefix::Tera => result << 40,
-                Prefix::Giga => result << 30,
-                Prefix::Mega => result << 20,
-                Prefix::Kilo => result << 10,
+                Prefix::Tera => result >> 40,
```
Comment (ammgws) [inline src/blocks/disk_space.rs:211]: Maybe it's better to leave `result` as bytes and shift the warning/alert values to bytes instead. For example, with this way a `result` of 4.6GB ends up becoming just `4`

Comment (MaxVerevkin) [reply src/blocks/disk_space.rs:211]: You are right. Or just convert to float and divide.

Comment (ammgws) [reply src/blocks/disk_space.rs:211]: Yeah either way

**Comment by ammgws** [comment]
Thanks!

**Pull request merged**